CI/flatpak/com.obsproject.Studio.json

```diff
@@ -236,6 +237,7 @@
         "cp -R ./libcef_dll_wrapper/libcef_dll_wrapper.a /app/cef/libcef_dll_wrapper"
```

Comment:
unrelated to the change at hand, but stripping the extension off the wrapper seems, questionable?

Reply:
or does it end up with a directory of the same name as the archive?

Reply:
Oh that's a folder name. The file is installed as `/app/cef/libcef_dll_wrapper/libcef_dll_wrapper.a` which is what the browser plugin expects